docs/tutorial.md

```diff
@@ -186,6 +186,66 @@ True
 
 ```
 
+### Class restrictions
+When working with OWL ontologies, you often need to look up or add class restrictions.
+A [restriction] restricts a class to only those individuals that satisfy the restriction.
+The Triplestore class has two convenient methods for this, that don't require knowledge about how restrictions are represented in RDF.
+They only support basic restrictions without any nested logical constructs.
+For more advanced restrictions, we recommend to use [EMMOntoPy] or [Owlready2].
+
+A restriction is described by the following set of parameters.
+
+  * **cls**: IRI of class to which the restriction applies.
+  * **property**: IRI of restriction property.
+  * **type**: The type of the restriction.  Should be one of:
+    - *some*: existential restriction (target is a class IRI)
+    - *only*: universal restriction (target is a class IRI)
+    - *exactly*: cardinality restriction (target is a class IRI)
+    - *min*: minimum cardinality restriction (target is a class IRI)
+    - *max*: maximum cardinality restriction (target is a class IRI)
+    - *value*: Value restriction (target is an IRI of an individual or a literal)
+
+  * **cardinality**: the cardinality value for cardinality restrictions.
+  * **value**: The IRI or literal value of the restriction target.
+
+For example. Lets assume that you have a class `onto:Bacteria` that you want to logically restrict to be unicellular.
+In Manchester syntax, this can be stated as `onto:Bacteria emmo:hasPart exactly 1 onto:Cell`.
+With tripper, you can state it with
```

Comment:
```suggestion
As an example, the class `onto:Bacteria` can be logically restricted to be unicellular.
In Manchester syntax, this can be stated as `onto:Bacteria emmo:hasPart exactly 1 onto:Cell`.
With Tripper this can be stated as:
```